diff --git a/node_modules/react-native-elements/src/input/Input.js b/node_modules/react-native-elements/src/input/Input.js
index e727fad..b28b153 100644
--- a/node_modules/react-native-elements/src/input/Input.js
+++ b/node_modules/react-native-elements/src/input/Input.js
@@ -76,6 +76,8 @@ class Input extends React.Component {
       labelStyle,
       labelProps,
       theme,
+      updateTheme,
+      replaceTheme,
       ...attributes
     } = this.props;

diff --git a/node_modules/react-native-elements/src/searchbar/SearchBar-default.js b/node_modules/react-native-elements/src/searchbar/SearchBar-default.js
index 39b7ebc..3712342 100644
--- a/node_modules/react-native-elements/src/searchbar/SearchBar-default.js
+++ b/node_modules/react-native-elements/src/searchbar/SearchBar-default.js
@@ -75,6 +75,7 @@ class SearchBar extends React.Component {
       showLoading,
       loadingProps,
       placeholderTextColor = theme.colors.grey3,
+      onClear,
       ...attributes
     } = rest;

diff --git a/node_modules/react-native-elements/src/text/Text.js b/node_modules/react-native-elements/src/text/Text.js
index b1b26c2..c738e34 100644
--- a/node_modules/react-native-elements/src/text/Text.js
+++ b/node_modules/react-native-elements/src/text/Text.js
@@ -17,6 +17,9 @@ const TextElement = props => {
     h2Style,
     h3Style,
     h4Style,
+    theme,
+    updateTheme,
+    replaceTheme,
     ...rest
   } = props;

